### ISSUE TITLE
delete TTS test Slack workspace - due ?

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#1146]
## Background Information

Broke out from https://github.com/18F/tts-tech-portfolio/issues/1076.

Once the gsa.gov [domain is claimed](https://slack.com/help/articles/115001379947-Claim-domains-on-Enterprise-Grid), Slack workspaces will all get added to the Grid. Once this is done, I don't think there will be much benefit to having a separate [test workspace](https://gsa-tts-testing.slack.com/), so we might as well get rid of it beforehand to have one less thing to migrate.

## Implementation Steps

- [ ] [Find out date of domain claim](https://gsa-tts.slack.com/archives/C01CVRM54S2/p1609262130011300?thread_ts=1609261365.011100&cid=C01CVRM54S2)
- [ ] Confirm all active use
   - Testing of [Charlie](https://github.com/18F/charlie) by @mgwalker 
- [ ] Get @mgwalker development credentials for Charlie

## Acceptance Criteria

- [ ] Workspace is deleted

**Comment by afeld** [comment]
Closing in favor of https://github.com/18F/tts-tech-portfolio/issues/1201.
